packages/plexus/src/LayoutManager/dot/convPlain.tsx

```diff
@@ -88,7 +88,8 @@ function parseGraph(str: string, startIndex: number): { end: number; graph: TLay
 function parseNode(str: string, startIndex: number) {
   // skip "node "
   const i = startIndex + 5;
-  const { value: key, end: keyEnd } = parseString(str, i);
+  const { value: raw, end: keyEnd } = parseString(str, i);
+  const key = raw.replace(/\\|\n/g, '');
```

Comment:
Maybe this should be sequestered into a function with a comment about long text IDs? An alternative is to use indexes for these keys, as a mapping, and map them back when parsing.